src/bottles/Bottle.java

```diff
@@ -74,25 +72,95 @@ void randomFillLevel(int capacity) {
         return bottle;
     }
 
-    // to improve
-    //static void bottleSettings() {
-    //    System.out.println("Bottle no " + bottle[i].getId() + ", capacity: " + bottle[i].getCapacity() + " l, Fill level: " + bottle[i].getFillLevel() + " l");
-    //}
+    static void bottleSettings(int howManyBottles, Bottle[] arrayName) {
+        for (int i = 0; i < howManyBottles; i++) {
```

Comment:
`arrayName.count` will be enough instead of `howManyBottles`(btw. `numberOfBottles` is much better name that `howManyBottles`).

Reply:
This parameter does not count anything. It stores int, received from user to initiate array length.

Reply:
Now I got what do You meant. Will be ok now?
```java
static void displayBottlesSettings(Bottle[] arrayName) {
        for (int i = 0; i < arrayName.length; i++) {
            System.out.println("Bottle no " + arrayName[i].getId() + ", capacity: " + arrayName[i].getCapacity() + " l, Fill level: " + arrayName[i].getFillLevel() + " l");
        }
    }
```

Reply:
Yeah, now it is better. 

Have you consider better name for `Bottle[] arrayName`?